spacy_llm/models/hf/falcon.py

```diff
@@ -22,7 +22,12 @@ def __init__(
         self._device: Optional[str] = None
         super().__init__(name=name, config_init=config_init, config_run=config_run)
         assert isinstance(self._tokenizer, transformers.PreTrainedTokenizerBase)
-        self._config_run["eos_token_id"] = self._tokenizer.eos_token_id
+        # self._config_run["eos_token_id"] = self._tokenizer.eos_token_id
```

Comment:
Let's remove the code line entirely, instead of just commenting it out. If we do want to leave it in, we should add another comment explaining why it's there & commented out.

Reply:
Forgot to remove it. Removed in https://github.com/explosion/spacy-llm/pull/222/commits/ae86e6d3a6d40146b4a2f390a1e377b4845eaf16.